src/lambda/expression.rs

```diff
@@ -35,25 +35,29 @@ pub enum Expression {
   Input(Box<Expression>, Vec<String>),
 }
 
-#[derive(Clone, Debug)]
+#[derive(Clone, Debug, Serialize)]
 pub enum Context {
   Value,
   Path(Vec<String>),
 }
 
-#[derive(Clone, Debug)]
+#[derive(Clone, Debug, Serialize)]
 pub enum Unsafe {
   Http {
+    #[serde(skip)]
     req_template: http::RequestTemplate,
     group_by: Option<GroupBy>,
+    #[serde(skip)]
     dl_id: Option<DataLoaderId>,
   },
+  #[serde(skip)]
   GraphQLEndpoint {
     req_template: graphql::RequestTemplate,
     field_name: String,
     batch: bool,
     dl_id: Option<DataLoaderId>,
   },
+  #[serde(skip)]
```

Comment:
We can't skip anything in that's an expression. We have two options here —
1. Move that non-serializable thing out of Expression, into a blueprint companion object that doesn't need to be serializable. Typically into ServerContext.
2. Implement Serialize/Deserialize into it.

Reply:
Managed to serialize Grahql and Http from this Enum, what I need to do now is to serialize grpc. The problem is that I need to serialize a `ProtobufOperation` that has 3 external types. I am blocked on this atm. Still looking for a solution.

Reply:
@meskill is there a way we can implement a custom serde for this ?

Reply:
oh, well, no simple way I think. 
The data used inside this is tied to a single `DescriptorPool` that is shared between many structures and use indexes and references to it. So, to proper serialize it we probably need to be able to recreate all that structure i.e. basically parse proto file again and create new ProtobufOperation.

Reply:
Dropping blueprint serialization from the scope of this PR.